CorsixTH/Lua/dialogs/menu.lua

```diff
@@ -636,12 +636,21 @@ function UIMenuBar:makeGameMenu(app)
   end
   options:appendCheckItem(_S.menu_options.lock_windows, boolean_runtime_config"lock_windows")
 
+  -- Edge Scrolling
   options:appendCheckItem(_S.menu_options.edge_scrolling,
     not app.config.prevent_edge_scrolling,
     function(item) app.config.prevent_edge_scrolling = not item.checked end,
```

Comment:
Not relevant to your PR, but I just noticed this toggle doesn't save, when the others do.

Reply:
I'll open another issue to look into it. It should be minor but deserves it's own testing to be sure we have got it working properly

Reply:
I've opened #1311 - I'll look into it another day as it looks like a simple job, but I suspect other buttons could also be broken too.

Reply:
Thanks, I was going to do this but got sleepy 👍